src/habitat.cr

```diff
@@ -226,12 +233,19 @@ class Habitat
         {% end %}
       {% end %}
 
+      {% # https://github.com/crystal-lang/crystal/pull/14490
+ if compare_versions(Crystal::VERSION, "1.13.0-dev") >= 0
```

Comment:
I need the `-dev` here otherwise the current nightly wouldn't get picked up. I'm hoping the final release is still `>=` the `-dev`.